src/cdk/a11y/interactivity-checker/interactivity-checker.spec.ts

```diff
@@ -449,35 +522,44 @@ describe('InteractivityChecker', () => {
 
       });
 
-    }));
+    });
 
-    describe('for iOS browsers', runIf(platform.IOS && platform.WEBKIT, () => {
+    describe('for iOS browsers', () => {
+      let shouldSkip: boolean;
 
-      it('should never allow div elements to be tabbable', () => {
-        let divEl = createFromTemplate('<div tabindex="0">', true);
+      beforeEach(() => {
+        shouldSkip = !platform.IOS && !platform.WEBKIT;
```

Comment:
Isn't that different in behavior? i.e. previously `IOS = true and WEBKIT = false` did not run, but now it will. Also now these tests run for any webkit-enabled browser right?

Reply:
You're right, I didn't flip the condition correctly.